Tests/TuistPluginTests/PluginServiceTests.swift

```diff
@@ -0,0 +1,134 @@
+import TSCBasic
+import TuistCore
+import TuistLoader
+import TuistLoaderTesting
+import TuistSupport
+import TuistSupportTesting
+import XCTest
+
+@testable import TuistPlugin
+
+final class PluginServiceTests: TuistTestCase {
+    private var mockModelLoader: MockGeneratorModelLoader!
+    private var mockGitHandler: MockGitHandler!
+    private var mockManifestFilesLocator: MockManifestFilesLocator!
+    private var subject: PluginService!
+
+    override func setUp() {
+        super.setUp()
+        do {
```

Comment:
I think it's more transparent to use `setUpWithError() throws` function and do not use the explicit `XCTFail` here